### DISCUSSION TITLE
sevi中level2的A,rho,sigma对一幅图里的电子是相同的吗

### DISCUSSION THREAD

**Discussion by meishuyao** [physics-data/faq#121] [Q&A] [answered]
与 level 1 类似，你也需要考虑白噪声，以及光电子不被探测的概率并在数据文件中标记；此外，针对**被探测**到的电子，你需要记录他们产生光斑的 $A$，$\sigma$, $\rho$，且在 `read_sim_data.py` 输出光电子个数、被探测到光电子个数之后**打印被探测到电子的这三个变量**。

如上加粗部分所示，看起来这三个变量对一幅图中所有电子是相同的。

另外，请问是不是每种mode要输出50000张图，而不是五万次实验输出为一张图。

**Comment by meishuyao** [comment]
@littlewu2508

**Comment by littlewu2508** [comment, marked as answer]
> 与 level 1 类似，你也需要考虑白噪声，以及光电子不被探测的概率并在数据文件中标记；此外，针对**被探测**到的电子，你需要记录他们产生光斑的 $A$，$\sigma$, $\rho$，且在 `read_sim_data.py` 输出光电子个数、被探测到光电子个数之后**打印被探测到电子的这三个变量**。
> 
> 如上加粗部分所示，看起来这三个变量对一幅图中所有电子是相同的。

每个电子的这三个变量都是不一样的。

> 
> 另外，请问是不是每种mode要输出50000张图，而不是五万次实验输出为一张图。

是的

**Reply by meishuyao** [reply]
就是说我在画图的时候需要打印约300个数据吗

**Reply by littlewu2508** [reply]
是的
